walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -218,46 +139,47 @@ In this task, you use the Identity Management service to protect both the client
 
 === Add Identity Management service to the client app (PWA) and create a realm for users
```

Comment:
actually if you read the title here, I think this section is just about creating the realm. The next section is actually about creating the clients. So we probably should move some of the content in this section to the next one, or create a different section to cover how to get KC config for the client app.